.github/workflows/versionControlAndAuditCheck.yml

```
@@ -0,0 +1,252 @@
+    runs-on: ubuntu-latest
+    env:
+      CONTINUE: false # makes sure that variable is correctly initialized in all cases
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get list of modified files by this PR
+        id: modified_files
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:28:28: Double quote to prevent globbing and word splitting

(shellcheck)